crew

```diff
@@ -209,30 +239,26 @@ end
 def resolveDependencies
   @dependencies = []
 
-  def pushDeps
-    if @pkg.dependencies
-      @dependencies.unshift @pkg.dependencies
-      
-      @pkg.dependencies.each do |dep|
-        search dep, true
-        pushDeps
+  if @pkg.dependencies
+    @pkg.dependencies.each do |dep|
+      if not search(dep, true)
+        if not @dependencies.include? dep
+          @dependencies.push(dep)
+        end
       end
     end
   end
-  
-  pushDeps
```

Comment:
@jam7: Thank you for the input.  We don't need packages.json but since I already spent the effort to create it, we have the data ready to pull from there to the ruby packages.  You are correct about the recursive dependencies.  My thinking was only prompt for dependency packages if they aren't already installed.  Currently, the process doesn't take that into account.  I'll keep working on it.